### PR TITLE
WIP: OrbTk book v. 1

### PR DESCRIPTION
**Pull request by FloVanGH**

### CODE REVIEW

File: manual/src/getting_started/run_examples.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Run Examples with cargo-node
+
+To run the examples on as browser, electron or cordova app you have to install
+
+```text
+cargo install -f cargo-node
+```
+
+Before you could use cargo node you have to install `npm` version 6.9.0. It is included in the `Node.js` version 10.16.3. You could download it from https://nodejs.org/dist/v10.16.3/.
```
Comment (Jonesey13) [inline manual/src/getting_started/run_examples.md]: "could" => "can" (it currently reads as if you can't do this anymore). Same problem with the root readme.

Comment (FloVanGH) [reply manual/src/getting_started/run_examples.md]: Ok thank you.

File: manual/src/getting_started/setup_rust.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Setup Rust
+
+OrbTk is an UI-Toolkit written in and for the programming language [Rust](https://www.rust-lang.org/). This guide wil help you to install Rust.
```
Comment (Fenex) [inline manual/src/getting_started/setup_rust.md]: wil -> will

File: manual/src/getting_started/setup_rust.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+If you are using Linux or macOS open up an terminal and copy and paste the text below and hit the enter key on your keyboard:
+
+```bash
+curl https://sh.rustup.rs -sSf | sh
+```
+
+## Install Rust on Windows
+
+Download an run the Rust windows installer from https://www.rust-lang.org/tools/install.
```
Comment (Fenex) [inline manual/src/getting_started/setup_rust.md]: an -> and

File: manual/src/getting_started/run_examples.md (hunk 1 above)
Comment (tzargoth) [reply manual/src/getting_started/run_examples.md:21]: "on as browser" --> "in a browser"

File: manual/src/getting_started/run_examples.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+To run the examples on as browser, electron or cordova app you have to install
+
+```text
+cargo install -f cargo-node
+```
+
+Before you could use cargo node you have to install `npm` version 6.9.0. It is included in the `Node.js` version 10.16.3. You could download it from https://nodejs.org/dist/v10.16.3/. 
+
+Rust's `cargo` is presumed. All other dependencies of cargo node will be installed automatic.
```
Comment (tzargoth) [inline manual/src/getting_started/run_examples.md:23]: automatic --> automatically

File: manual/src/getting_started/run_examples.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+You can start the widgets example by executing the following command:
+
+```text
+cargo run --example widgets --release
+```
+
+## Run Examples with cargo-node
+
+To run the examples on as browser, electron or cordova app you have to install
```
Comment (tzargoth) [inline manual/src/getting_started/run_examples.md:15]: "on as browser" --> "in a browser"

File: manual/src/welcome.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Welcome
+
+**OrbTk is an UI-Toolkit to write scalable cross platform graphical user interface in the programming language Rust.**
+
+We are glad you are here and interested in OrbTk. The aim of this manual is to help you with your first steps with OrbTk. 
+It will guide you step by step how to write you first applications with OrbTk and will help you to understand the core concepts
+of OrbTk.
+
+*Hint:* This manual is work in progress and more content will produced step by step.
```
Comment (tzargoth) [inline manual/src/welcome.md:9]: "manual is work" --> "manual is a work"; "content will produced" --> "content will be produced"

File: manual/src/getting_started/create_project.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+# Create project
+
+To start your first OrbTk project you could use [cargo](https://doc.rust-lang.org/cargo/) to create and run it.
```
Comment (tzargoth) [inline manual/src/getting_started/create_project.md:3]: could --> can

File: manual/src/getting_started/create_project.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+   name = "my_project"
+   version = "0.1.0"
+   authors = ["m_name <my.name@my_mail.org>"]
+   edition = "2018"
+   
+   [dependencies]
+   orbtk = "0.3.1-alpha1"
+   ```
+
+4. As next replace the content of the file `main.rs` with the following content:
```
Comment (tzargoth) [inline manual/src/getting_started/create_project.md:30]: "As next" --> "Next, "

File: manual/src/getting_started/setup_rust.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+## Install Redoxer (Redox OS)
+
+If you want build and run your Rust application on a [KVM](https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine) capable OS for Redox you could use [redoxer](https://gitlab.redox-os.org/redox-os/redoxer).
```
Comment (tzargoth) [inline manual/src/getting_started/setup_rust.md:19]: could --> can

File: manual/src/getting_started/setup_rust.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+To install Redoxer you have first install the rust toolchain. After that open up an terminal and copy and paste the text below and hit the enter key on your keyboard:
```
Comment (tzargoth) [inline manual/src/getting_started/setup_rust.md:21]: "have first" --> "have to first"

File: manual/src/welcome.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+
+### [Getting Started](getting_started/getting_started.md)
+
+This section will guide you how to setup the whole Rust toolchain and how to create your first OrbTk project.
```
Comment (tzargoth) [inline manual/src/welcome.md:13]: "you how" --> "you through the process of how"; "how to create" --> "create"

File: manual/src/welcome.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+In this section you will find different tutorials that will guide you step by step how to build different applications with OrbTk.
+
+### Showcases
+
+* [Space Editor](https://codeberg.org/pinhead-galaxy/space-editor): 2D Tile Map Editor compatible with OrbGame
+* [OrbCalculator](https://gitlab.redox-os.org/redox-os/orbcalculator): Calculator based on OrbTk
+
+### Contribution
+
+If you want to help bring OrbTk further or you have feedback check our issue board https://gitlab.redox-os.org/redox-os/orbtk/-/boards/120. You could also discuss with us about OrbTk on the Redox chat https://redox-os.org/community/ (join the orbital channel).
```
Comment (tzargoth) [inline manual/src/welcome.md:26]: "could also discuss with us about OrbTk" --> "can also share ideas with us or ask question about OrbTk"

**Comment by FloVanGH** [comment]
@tzargoth thanks for your feedback

**Pull request merged**